staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -107,6 +107,15 @@ func MonitorRequest(request *http.Request, verb, resource, subresource, scope, c
 		}
 	}
 
+	// make get pod log with follow option api request reported as watch
+	if verb == "GET" && resource == "pods" && subresource == "log" {
```

Comment:
@sttts Then we need to dynamicly add different label for different request? how about adding a new label called query_params and just store all the querys in it, like query_params="watch=false&follow=true&detail=false".I think this way is simpler